src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  --colorAction: #00acdc;
+}
+
+body,
+ul {
+  margin: 0;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  margin: auto 0;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
```

Comment:
you already reset it for the ul selector

Reply:
thanks for the comment. resolved in this commit https://github.com/mate-academy/layout_moyo-header/pull/2452/commits/847767bd70c6334cd87457d2cf71a62af07b42e9